code/game/gamemodes/changeling/powers/tiny_prick.dm

```diff
@@ -54,189 +67,96 @@
-	feedback_add_details("changeling_powers","ED")
-	return 1
-
 /obj/effect/proc_holder/changeling/sting/mute
 	name = "Mute Sting"
 	desc = "We silently sting a human, completely silencing them for a short time."
-	helptext = "Does not provide a warning to the victim that they have been stung, until they try to speak and cannot."
+	helptext = "Our target will not be alerted to their silence until they attempt to speak and cannot."
 	sting_icon = "sting_mute"
 	chemical_cost = 20
-	dna_cost = 2
+	dna_cost = 3
 
 /obj/effect/proc_holder/changeling/sting/mute/sting_action(mob/user, mob/living/carbon/target)
 	add_logs(user, target, "stung", "mute sting")
-	target.silent += 30
+	target.reagents.add_reagent("mutetoxin", 5)
 	feedback_add_details("changeling_powers","MS")
 	return 1
 
 /obj/effect/proc_holder/changeling/sting/blind
 	name = "Blind Sting"
-	desc = "Temporarily blinds the target."
-	helptext = "This sting completely blinds a target for a short time."
+	desc = "We inject a serum that attacks and damages the eyes."
+	helptext = "The victim will immediately be blinded for a short time in addition to becoming permanently nearsighted."
 	sting_icon = "sting_blind"
 	chemical_cost = 25
-	dna_cost = 1
+	dna_cost = 2
 
 /obj/effect/proc_holder/changeling/sting/blind/sting_action(mob/user, mob/target)
 	add_logs(user, target, "stung", "blind sting")
-	target << "<span class='danger'>Your eyes burn horrifically!</span>"
+	target << "<span class='userdanger'>Your eyes burn horrifically!</span>"
 	target.disabilities |= NEARSIGHT
 	target.eye_blind = 20
 	target.eye_blurry = 40
+	target.confused += 5 //Going instantaneously blind often makes one a little disoriented
 	feedback_add_details("changeling_powers","BS")
 	return 1
 
-/obj/effect/proc_holder/changeling/sting/LSD
-	name = "Hallucination Sting"
-	desc = "Causes terror in the target."
-	helptext = "We evolve the ability to sting a target with a powerful hallucinogenic chemical. The target does not notice they have been stung, and the effect occurs after 30 to 60 seconds."
-	sting_icon = "sting_lsd"
-	chemical_cost = 10
-	dna_cost = 1
-
-/obj/effect/proc_holder/changeling/sting/LSD/sting_action(mob/user, mob/living/carbon/target)
-	add_logs(user, target, "stung", "LSD sting")
-	spawn(rand(300,600))
-		if(target)
-			target.hallucination = max(400, target.hallucination)
-	feedback_add_details("changeling_powers","HS")
-	return 1
-
 /obj/effect/proc_holder/changeling/sting/cryo
 	name = "Cryogenic Sting"
 	desc = "We silently sting a human with a cocktail of chemicals that freeze them."
-	helptext = "Does not provide a warning to the victim, though they will likely realize they are suddenly freezing."
-	sting_icon = "sting_cryo"
+	helptext = "This will provide an ambiguous warning to the victim after a short time."
+	sting_icon = "sting_cryogenic"
 	chemical_cost = 15
-	dna_cost = 2
+	dna_cost = 3
+	conscious_req = 0 //Can be used on the unconscious

```

Comment:
>tweak: All stings no longer function on the dead. Cryogenic Sting now only works on conscious targets.

This should be set to 1 and the default (at line 6) should be set to 0 if I'm reading this right (which I probably am not)